<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgGroupType</class>
 <widget class="QDialog" name="DlgGroupType">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select DataSet type</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>IED:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_ied_name">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 85, 255);
font: 75 12pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget_type">
     <column>
      <property name="text">
       <string>LDevice</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>DataSet</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>OK(&amp;O)</string>
       </property>
       <property name="icon">
        <iconset resource="oms_hmi.qrc">
         <normaloff>:/image/ok</normaloff>:/image/ok</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Cancel(&amp;C)</string>
       </property>
       <property name="icon">
        <iconset resource="oms_hmi.qrc">
         <normaloff>:/image/cancel</normaloff>:/image/cancel</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="oms_hmi.qrc"/>
 </resources>
 <connections/>
</ui>
